persistence/sql/migratest/testdata/20220301102701_testdata.sql

```diff
@@ -1 +1,6 @@
-INSERT INTO identity_credentials (id, nid, config, identity_credential_type_id, identity_id, created_at, updated_at, version) VALUES ('4cefc264-4291-4abc-8f26-cc0217874f14', '884f556e-eb3a-4b9f-bee3-11345642c6c0', '{"hashed_password":"$argon2id$v=19$m=131072,t=2,p=1$lQFPaKxXqPL56/mU7vRi4w$6aldHyBnURt8sP8+xu41Ng"}', '22bff9ae-f5aa-45d7-803b-97ec0b4e7b32', '5ff66179-c240-4703-b0d8-494592cefff5', '2013-10-07 08:23:19', '2013-10-07 08:23:19', 0);
+INSERT INTO identity_credentials (id, nid, config, identity_credential_type_id, identity_id, created_at, updated_at,
+                                  version)
+VALUES ('4cefc264-4291-4abc-8f26-cc0217874f14', '884f556e-eb3a-4b9f-bee3-11345642c6c0',
```

Comment:
The code changes actually found that this migration was incorrect. The identity now had two password credentials, which doesn't make sense. By making this a TOTP credential, the new code no longer complained.